configs/sites/tier1/aws-pcluster/README.md

```diff
@@ -383,7 +383,8 @@ git clone --recurse-submodules -b release/1.7.0 https://github.com/JCSDA/spack-s
 cd spack-stack-1.7/
 . setup.sh
 spack stack create env --site aws-pcluster --template=unified-dev --name=unified-intel
-spack env activate -p envs/unified-env
+cd unified-intel
```

Comment:
that can't be - `unified-intel` is created under `envs`

Reply:
done